itl/command-plugins.conf

```diff
@@ -580,7 +580,6 @@ object CheckCommand "http" {
 		}
 	}
 
-	vars.http_address = "$check_address$"
```

Comment:
That must not be removed.

Reply:
Fair enough, so how do you make vhost checks work when there is always an IP address included as well.  Is there an elegant way to use a set_if on the `-I` option to not include it when `$http_vhost$` is set?

Ill fix it the best way, just need some advice, and it does need fixing. 

Thanks.